HISTORY.md

```diff
@@ -1,3 +1,21 @@
+v0.5.0
```

Comment:
@ramirezdiego , worth adding the data to the title of the release like in https://github.com/XENONnT/straxen/blob/master/HISTORY.md?

Reply:
Ok, will do. Was respecting the old format, but I guess this helps for PyPi visualization.

Reply:
Many thanks, I did the same previously but only realized that I omitted it later